CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.17
+* Return changes from 0.6.14

```

Comment:
Could you please change this to `Return changes from 0.6.14-0.6.15` as more correct variant?